bpf/bpf_lxc.c

```diff
@@ -523,6 +523,9 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 	struct ct_state ct_state = {};
 	__be32 orig_dip;
 	__u32 tunnel_endpoint = 0;
+#ifdef ENABLE_VTEP
+	mac_t vtep_mac = 0;
```

Comment:
We usually just use `__maybe_unused` to avoid having too many preprocessor macros in the variable initializations:
```suggestion
	mac_t vtep_mac __maybe_unused = 0;
```

Reply:
do you mean I should also remove the `#ifdef ENABLE_VTEP` ?

Reply:
@joestringer not sure rocket Rocket Emoji meaning here, I guess is to remove `ifdef` :)

Reply:
yeah that's what I mean :) :rocket:

Reply:
done